beamer/events.py

```diff
@@ -268,6 +274,14 @@ def __init__(
         self._event_abis = _make_topics_abi_mapping_for_contracts(contracts)
         self._confirmation_blocks = confirmation_blocks
         self._log = structlog.get_logger(type(self).__name__).bind(chain_id=self._chain_id)
+        self._log = structlog.get_logger(type(self).__name__)
+        self._ws_rpc_url = re.sub("^http", "ws", web3.provider.endpoint_uri)
+        # FIXME - add "address" filter
```

Comment:
At the moment we are subscribed to all logs. Should add filtering by address. 

```
addresses=[
    '0xb51722efe9cD480EC416AE455B68Ed36829ca089',   # Beamer contract
    '0x6b6AfE68b3083aE37E7FE91482aEc7bD7267226B',   # Beamer contract
]
{"id": 1, "method": "eth_subscribe", "params": ["logs", {"address": addresses}]}
```

There is an issue I don't understand. Dont receive any logs after I am subscribed to the contracts addresses.
Recognised I am receiving updates on `0xf9d2553991c938c821c3242a81f87958a947e1b5` but have no idea where does it come from.

This works:
```
addresses=[
    '0xf9d2553991c938c821c3242a81f87958a947e1b5',   # Artifical contract...
    '0xb51722efe9cD480EC416AE455B68Ed36829ca089',   # Beamer contract
    '0x6b6AfE68b3083aE37E7FE91482aEc7bD7267226B',   # Beamer contract
]
{"id": 1, "method": "eth_subscribe", "params": ["logs", {"address": addresses}]}
```